communication/src/main/java/org/jboss/da/communication/aprox/impl/AproxConnectorImpl.java

```diff
@@ -82,6 +85,7 @@
             GraphExport export = mod.graph(req);
 
             if (export == null || export.getRelationships() == null) {
+                // no dependencies found
```

Comment:
If this mean that the GAV exists but has no dependencies, there shouldn't be ```return Optional.empty()```, but ``` return Optional.of(new GAVDependencyTree(gav))```.